Source/DafnyCore/Auditor/AuditReport.cs

```diff
@@ -115,13 +115,21 @@ public class AuditReport {
     StringBuilder text = new StringBuilder();
 
     foreach (var module in modulesWithEntries) {
-      text.AppendLine($"# Module `{module.Name}`");
+      if (module.Name.Equals("_module")) {
```

Comment:
A Name comparison seems like a frail method to determine whether it's the default module.

Reply:
So at least use a const value somewhere, or an IsRootModule predicate or something.

Reply:
Yeah, I strongly agree. I had initially planned to make this PR as minimal as possible, and this sort of string comparison is what the rest of the code does, but it wouldn't be hard to introduce a reusable predicate as part of this PR.

Reply:
Oh, and it turns out that this and the related class property already existed, and I hadn't found them before because they do better than using string comparison.